README.adoc

```diff
@@ -1860,6 +1860,19 @@ Time.zone.now + 2.days
 2.days.from_now
 ----
 
+Use an instance of `ActiveSupport::Duration` as a receiver to calculate relative time.
+
+[source,ruby]
+----
+# bad - It's confusing to read
+created_at.after(1.minute)
+expiration_date.before(3.days)
```

Comment:
since/in/ago?
Where are before/after defined?

Reply:
Thank you for the review, I have fixed the issue at a451ad159c704783060866351a86848dfb5ff8ad.